modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java

```diff
@@ -370,6 +373,172 @@ public void testObjectEntry1toMObjectRelatedModelsProviderImpl()
 
 		_objectRelationshipLocalService.deleteObjectRelationship(
 			objectRelationship);
+
+		_setUser(TestPropsValues.getUser());
+
+		ObjectDefinition objectDefinitionA =
+			_objectDefinitionLocalService.addCustomObjectDefinition(
+				TestPropsValues.getUserId(), false,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				"C" + RandomTestUtil.randomString(), null, null,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				ObjectDefinitionConstants.SCOPE_COMPANY,
+				ObjectDefinitionConstants.STORAGE_TYPE_DEFAULT,
+				Arrays.asList(
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING,
+						RandomTestUtil.randomString(), StringUtil.randomId())));
+
+		objectDefinitionA =
+			_objectDefinitionLocalService.publishCustomObjectDefinition(
+				TestPropsValues.getUserId(),
+				objectDefinitionA.getObjectDefinitionId());
```

Comment:
@pedro-oliveira446 can we use _objectDefinition1 instead of objectDefinitionA?

Reply:
Yes, I managed to do it this way. 
@MarinhoFeliphe